taskchampion/src/storage/test.rs

```diff
@@ -273,18 +273,17 @@ pub(super) fn delete_task_exists(mut storage: impl Storage) -> Result<()> {
 }
 
 pub(super) fn get_pending_tasks(mut storage: impl Storage) -> Result<()> {
-    let uuid1 = Uuid::new_v4();
-    let uuid2 = Uuid::new_v4();
+    let uuid1 = Uuid::new_v4(); // in working set
+    let uuid2 = Uuid::new_v4(); // in working set
+    let uuid3 = Uuid::new_v4(); // in working set but does not exist
+    let uuid4 = Uuid::new_v4(); // not in working set but exists= Uuid::new_v4();
```

Comment:
Looks like a bit of extra text got included in that comment.

Reply:
This is now fixed :smile: